guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+>
+> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
+>
+> Cause and remedy might occur in combinations other than pairs. It is possible to have:
+>
+> - Multiple causes with the same remedy
+> - A single cause with more than one remedy
+> - A remedy with no known cause
+> - A cause with no known remedy
+> The troubleshooting information type also can be used to document alarm clearing strategies.
+
+You can use the [troubleshooting topic template](../templates/resources/troubleshooting.md).
+
+### Release Notes
+
+Announce what's new in Kyma.
+
+After an introductory paragraph that outlines the city that's the namesake of the current release, a list briefly presents the new and changed features. Links lead to longer paragraphs that describe the changes in more detail.
+
+## Graphics
+
+### Architecture and Flow Diagrams
+
+For information about our diagram style, see [Diagrams](../02-diagrams.md).
+
+### Screenshots
+
+Wherever possible, present screenshots as simplified user interfaces (SUI). Basically, this means blurring out all UI elements that aren't essential for the task at hand.
+
+For more information, see:
+• https://www.techsmith.com/blog/simplified-user-interface/
+• https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/
+
+## Target Groups
+
+The general assumption is that the audience is familiar with the following terms and does not require the explanation of technical concepts behind them:
+
+- Kubernetes
+- Docker and containers
+
+<!-- Information about generic target groups is based on the SAP Styleguide: https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/546dfb06e80c4005aabc4795d548fe35.html -->
+
+### Decision maker
+
+**Tasks**: Assesses the software to make sure that it meets the company's needs. Requires the facts – not just marketing spin – before signing on the dotted line.
+**Goal**: Purchase the right solution for the company, and ensure stakeholders back this decision.
+
+### Software developer
+
+**Tasks**: Interested in technical topics. Solid knowledge of programming languages. Experienced in programming and development projects. Expert in the technical or business area. Uses and contributes to community content.
+**Goal**: Develop and enhance software.
+
+### Admin/Operations
+
+**Tasks**: Deals with installation, upgrades, system troubleshooting.
+**Goal**: Support the ongoing operations and evolution of the Kyma implementation.
+
+## Documentation Structure
+
+On the Kyma website, we have five main tabs containing multiple documents each, plus a glossary.
+
+### Overview
+
+**Target Group**: Decision Makers (Tech Leads) and newbies.
```

Comment:
Here I'd also separate `Target group` from the description and put them in separate lines:

<img width="1042" alt="Screen Shot 2021-08-10 at 13 26 24" src="https://user-images.githubusercontent.com/35192450/128859018-64e2a071-c43c-4bd0-bba1-eef50299aa28.png">

Reply:
I'm trying to fix the line break with extra lines between the sentences *fingerscrossed